src/main/resources/db/migration/V0_0_1_20240403234120__insert-an-user-into-users-table.sql

```diff
@@ -0,0 +1,12 @@
+INSERT INTO
+  `users` (
+    `username`,
+    `password`,
+    `email_address`
+  )
+VALUES
+  (
+    'danh',
```

Comment:
Please update this user information to something general like 'admin/password' and add it to the README file as well.